client/src/views/HomeView.vue

```diff
@@ -1,108 +1,72 @@
 <template>
   <div class="home">
     <!-- Login Button -->
-    <v-btn
-      elevation="2"
-      @click="$router.push('/login')"
-    >
-      LOGIN
-    </v-btn>
-    <br><br><br>
+    <v-btn elevation="2" @click="$router.push('/login')"> LOGIN </v-btn>
+    <br /><br /><br />
 
     <!-- Search criteria -->
-    <v-col
-      id="country-box"
-      cols="12"
-      sm="6"
-      md="3"
-    >
-      <v-text-field
-          label="Country"
-          v-model="country"
-          @keyup.enter="search()"
-      >
-      </v-text-field>
-    </v-col>
-
-    <v-col
-      id="city-box"
-      cols="12"
-      sm="6"
-      md="3"
-    >
-      <v-text-field
-          label="City"
-          v-model="city"
-          @keyup.enter="search()"
-      >
-      </v-text-field>
-    </v-col>
+    <div id="loc-box">
+      <GmapAutocomplete @place_changed="setPlace"></GmapAutocomplete>
+    </div>
 
-    <v-col
-      id="budget-box"
-      cols="12"
-      sm="6"
-      md="3"
-    >
-      <v-text-field
-          label="Budget"
-          v-model="budget"
-          @keyup.enter="search()"
-      >
+    <v-col id="budget-box" cols="12" sm="1" md="1">
+      <v-text-field label="Budget" v-model="budget" @keyup.enter="search()">
       </v-text-field>
     </v-col>
 
     <!-- If Invalid Search -->
     <p v-if="validSearch === false" id="invalid-search-error">Invalid Search</p>
-    <br>
+    <br />
 
     <!-- Search button -->
-    <v-btn
-        elevation="2"
-        @click="search()"
-    >
-      Search
-    </v-btn>
+    <v-btn elevation="2" @click="$router.push('/search')"> Search </v-btn>
   </div>
 </template>
 
 <script>
 // @ is an alias to /src
 
-import SearchHandler from '../classes/SearchHandler.js';
+import Storage from "../classes/Storage.js";
+import Location from "../classes/Location.js"
 
 export default {
-  name: 'HomeView',
-  components: {
-
-  },
-  data: ()=> {
+  name: "HomeView",
+  components: {},
+  data: () => {
     return {
-      country: "",
-      city: "",
+      place: "",
```

Comment:
I recommend commenting a short sentence on the uses of each property in your SFCs state. In its current form the property names may be undescriptive just by themselves